### ISSUE TITLE
[FR] When selecting a fractal in the "Settings" toolbar, it should probably also execute a "Render" to the viewport

### ISSUE THREAD

**Issue by benwinding** [buddhi1980/mandelbulber2#810]
##### System information (version)
- mandelbulber2 => 2.20-1build1
- Operating System / Platform => Ubuntu 64 Bit
- From Ubuntu Store

##### Detailed description
When selecting a setting from the top "Settings Toolbar" it shows a blank screen. You then need to either click render, or zoom to get the fractal to show. See the following screen capture:

It would be much more intuitive if the scene would *automatically* render, when you select one of these settings.
![ezgif-7-e81e28959b10](https://user-images.githubusercontent.com/11782590/93032773-6508c680-f672-11ea-9ffc-16a426cd9c56.gif)

The `auto-refresh` checkbox is also _not_ persisted when selecting different fractals from settings...

##### Steps to reproduce
1. Open `Mandelbulber v2`
2. Select a default fractal from the "Setting Toolbar" at the top
3. .... Nothing is shown in viewport

##### Thank you
Anyway, thank you to the developers and contributors of this beautiful project. I feel lucky to have discovered it!

![image](https://user-images.githubusercontent.com/11782590/93033296-d184c500-f674-11ea-8fad-decfc8a70841.png)
_[ My attempt at love hearts ]_

**Comment by buddhi1980** [comment]
For both mentioned problems there were requests from users to do opposite reaction.
Sometimes users stores settings on toolbar which are heavy to render and they don't want to render them automatically.
Auto-refresh is disabled from the same reason. Loading of parameters changes parameters, so the image would be rendered automatically. There was also another problem. Loading of new settings during auto-refreshing sometimes caused mess in the settings (settings were loaded partially).

**Comment by benwinding** [comment]
> For both mentioned problems there were requests from users to do opposite reaction.

Yes, I had a feeling that might be the case... but still as a new user it's a little unintuitive, and I've used plenty of CAD software and was considering giving up on this software, I think many people would probably give up before a single render...

Ideally I would sugguest the default setting to render on settings change. 
But perhaps a simple notification _"Click to Render"_ to encourage the user down the happy path (as shown below):

![image](https://user-images.githubusercontent.com/11782590/93146750-ce4e0f80-f72e-11ea-9691-4dd1ba8cb98c.png)

The reason I created this ticket is that it's clear that the developers have empathy for the Users, as there's many clear warnings, well thought out UI and great documentation. 

I just fear many people won't even give it a chance if they think the application is broken...
